mongodbforms/documents.py

```diff
@@ -357,7 +358,7 @@ def __new__(cls, name, bases, attrs):
         return new_class
 
 
-class BaseDocumentForm(BaseForm):
+class BaseDocumentForm(BaseModelForm):
```

Comment:
have you verified that using the admin works? adding documents, editing documents, deleting documents, searching documents, etc...?